fs/accounting/stats.go

```diff
@@ -374,7 +385,7 @@ func (s *StatsInfo) String() string {
 			_, _ = fmt.Fprintf(buf, "Transferred:   %10d / %d, %s\n",
 				s.transfers, ts.totalTransfers, percent(s.transfers, ts.totalTransfers))
 		}
-		_, _ = fmt.Fprintf(buf, "Elapsed time:  %10ss\n", strings.TrimRight(elapsedTime.Truncate(time.Minute).String(), "0s")+fmt.Sprintf("%.1f", elapsedTimeSecondsOnly.Seconds()))
+		_, _ = fmt.Fprintf(buf, "Elapsed time:  %10s\n", elapsedTime.String())
```

Comment:
This looks like an unrelated change. It will cause the elapsed time to go from `10s` to `10.5s` to `11s` which is visually distracting (and the reason for the original code - though the original code is pretty weird now I look at it!)

Reply:
Actually, the original code did that too, I just removed the unnecessary splitting between minutes/seconds that was happening (time.Duration.String() does that just fine). I was actually going to remove tenths-of-second (and only show seconds), but didn't figure it was my place to decide that.

Reply:
This is what the original code does

```
$ rclone copy -v --stats 0.5s --bwlimit 1k ~/go/src/github.com/rclone/rclone /tmp/rclone.copy 2>&1 | grep Elapsed
Elapsed time:         0.5s
Elapsed time:         1.0s
Elapsed time:         1.5s
Elapsed time:         2.0s
Elapsed time:         2.5s
Elapsed time:         3.0s
Elapsed time:         3.5s
Elapsed time:         4.0s
```

And this is what your code does

```
$ rclone copy -v --stats 0.5s --bwlimit 1k ~/go/src/github.com/rclone/rclone /tmp/rclone.copy 2>&1 | grep Elapsed
Elapsed time:          0s
Elapsed time:          1s
Elapsed time:          1s
Elapsed time:          2s
Elapsed time:          2s
Elapsed time:          3s
Elapsed time:          3s
Elapsed time:          4s
```

So it seems to have removed the tenths?